jinja2/includes/icons/emojis/thumbs-down.svg

```diff
@@ -1 +1 @@
-<svg class="icon icon-emoji-thumbs-down" width="25" height="25" xmlns="http://www.w3.org/2000/svg" aria-hidden="true"><text transform="matrix(1 0 0 -1 -15 38)" fill="#262D2D" fill-rule="evenodd" font-family="AppleColorEmoji, Apple Color Emoji" font-size="25"><tspan x="15" y="36">👍</tspan></text></svg>
+<svg viewBox="0 0 914.286 1024" class="icon icon-thumbs-down"><path class="path0" stroke="inherit" d="M146.286 329.143c0 20-16.571 36.571-36.571 36.571-20.571 0-36.571-16.571-36.571-36.571 0-20.571 16-36.571 36.571-36.571 20 0 36.571 16 36.571 36.571zm91.428 292.571V256c0-20-16.571-36.571-36.571-36.571H36.572C16.572 219.429.001 236 .001 256v365.714c0 20 16.571 36.571 36.571 36.571h164.571c20 0 36.571-16.571 36.571-36.571zm645.143-85.143c19.429 21.714 31.429 54.857 31.429 85.143-.571 59.429-50.286 109.714-109.714 109.714H646.286c4.571 18.286 10.286 24 16.571 36.571 14.857 29.714 32 62.857 32 109.714 0 44 0 146.286-128 146.286-9.714 0-18.857-4-25.714-10.857-24.571-24-31.429-59.429-37.714-93.143-6.857-33.143-13.143-67.429-35.429-89.714-17.714-17.714-37.143-42.286-57.714-68.571-25.143-33.143-80-101.143-101.143-102.857-18.857-1.714-34.857-17.714-34.857-36.571V256c0-20 17.143-36 36.571-36.571 20-.571 54.286-12.571 90.286-25.143 61.714-21.143 138.857-48 220.571-48h73.714c50.286.571 88 15.429 112.571 44.571 21.714 25.714 31.429 60.571 28 103.429C850.285 308 861.142 326.857 866.856 348c6.286 22.857 6.286 45.714 0 66.857 17.143 22.857 25.714 49.714 24.571 78.286 0 8-2.286 25.143-8.571 43.429z"/></svg>
```

Comment:
Github says this SVG is invalid - does it work for you?

Also, does it need "aria-hidden"? Or should that be removed from ``thumbs-up.svg``?

Reply:
They worked for me locally :/ I also have ru them through SVGOMG and it also did not complain. I missed the `aria-hidden`, my bad.